modules/flowable-job-service/src/main/java/org/flowable/job/service/event/impl/FlowableJobEventBuilder.java

```diff
@@ -67,8 +67,13 @@ protected static void populateEventWithCurrentContext(FlowableEngineEventImpl ev
             if (persistedObject instanceof Job) {
                 Job jobObject = (Job) persistedObject;
                 if (jobObject.getScopeType() == null) {
-                    event.setExecutionId(jobObject.getExecutionId());
-                    event.setProcessInstanceId(jobObject.getProcessInstanceId());
+                    if ("async-complete-call-actiivty".equals(jobObject.getJobHandlerType())) {
```

Comment:
Wouldn't it be better to use the public static variable `AsyncCompleteCallActivityJobHandler.TYPE` instead of a literal string?      At least it would hide the incorrect spelling of `activity` :smile:

Reply:
> Wouldn't it be better to use the public static variable `AsyncCompleteCallActivityJobHandler.TYPE` instead of a literal string? At least it would hide the incorrect spelling of `activity` 😄

Hi @dbmalkovsky, thanks for your feedback. The typo in existing flowable code is quite unfortunate... excellent intention to cover it...
However, the change here is in FlowableJobEventBuilder which is located in maven module flowable-job-service, from here the AsyncCompleteCallActivityJobHandler class (and its constant) is not reachable as it is located in flowable-engine without dependency in that direction.